include/AutomationClip.h

```diff
@@ -92,20 +92,8 @@ class LMMS_EXPORT AutomationClip : public Clip
 	TimePos timeMapLength() const;
 	void updateLength();
 
-	TimePos putValue(
-		const TimePos & time,
-		const float value,
-		const bool quantPos = true,
-		const bool ignoreSurroundingPoints = true
-	);
-
-	TimePos putValues(
-		const TimePos & time,
-		const float inValue,
-		const float outValue,
-		const bool quantPos = true,
-		const bool ignoreSurroundingPoints = true
-	);
+	TimePos putValue(TimePos time, float value, tick_t clear = 0);
+	TimePos putValues(TimePos time, float inValue, float outValue, tick_t clear = 0);
```

Comment:
Removed `const` from all these, is that correct?

Reply:
Yes - top-level `const` and `volatile` have no effect in function declarations, so should be omitted to reduce noise.